homeassistant/components/risco/services.py

```diff
@@ -0,0 +1,41 @@
+"""Services for Risco integration."""
+
+from datetime import datetime
+
+import voluptuous as vol
+
+from homeassistant.core import HomeAssistant, ServiceCall
+from homeassistant.helpers import config_validation as cv
+
+from .const import CONF_CONFIG_ENTRY_ID, DOMAIN, SERVICE_SET_TIME
+from .models import LocalData
+
+ATTR_TIME = "time"
+
+
+async def async_setup_services(hass: HomeAssistant) -> None:
+    """Create the Risco Services/Actions."""
+
+    async def _set_time(service_call: ServiceCall) -> None:
+        config_entry_id = service_call.data.get(CONF_CONFIG_ENTRY_ID, None)
+        time = service_call.data.get(ATTR_TIME, None)
+
+        time_to_send = time
+        if time is None:
+            time_to_send = datetime.now()
```

Comment:
Are you sure the test is running correctly? Codecov is complaining this line isn't covered.

Reply:
You're right. I've added an additional test to cover it.